tests/test_snapshots.py

```
@@ -0,0 +1,1530 @@
+    orch.client_app_downloader.handle_snapshots = Mock(return_value=release)
+    orch.client_app_downloader.get_snapshot_version_code = Mock(return_value=100)
+    orch.client_app_downloader.should_process_snapshot = Mock(return_value=True)
+    selected = [release.assets[0]]
+    orch.client_app_downloader.get_selected_snapshot_assets = Mock(
+        return_value=selected
+    )
+    orch.client_app_downloader.download_snapshot_asset = Mock(
+        return_value=DownloadResult(
+            success=True,
+            release_tag="snapshot",
+            file_path="/tmp/x.apk",
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/x.apk"

(S108)